scss/application.scss

```
@@ -1,2 +1,8 @@
 @import 'equalizer';
+@import '_colors';
 @import 'styles';
+@import 'detail';
+
+* {
+  font-family: 'Roboto';
```

Add second font for good practice 
```suggestion
  font-family: 'Roboto', sans-serif;
```